src/Transition/OutputRulesFiles/OutputChanges22-1-0-to-22-2-0.md

```diff
@@ -66,3 +66,29 @@ The Adaptive Comfort Summary/Report was renamed to match the HTML:
 ```
 
 See pull request [#9461](https://github.com/NREL/EnergyPlus/pull/9461) for more details.
+
+### Latent Sizing
+
+New report variables were added to identify results associated with latent sizing.
+
+    Zone Air System Latent Heating Energy
```

Comment:
These aren't sizing results, are they? It looks like their simulation results?

Reply:
Reworded this output changes doc.